pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "vame-py"
-version = '0.1.0'
+version = '0.0.2'
```

Comment:
Usually the versions start at 0.1.0. See semantic versioning: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
Leaving it as patch just for forcing first realase on pypi (I can't get a token for a specific repository without releasing a version).
Should I use 0.1.0 even without being stable ? I was wondering to push 0.1.0 once we merge to main.

Reply:
I don't think we should be releasing from dev in the first place. Let's wait for a meeting before we publish on pupils